.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,4 +24,4 @@ PR flow tips:
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26806&commitId=ab2adcd5fd009fe4723adfbd8dce2e12b3ea938e&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1847 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|1| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>